articles/active-directory/app-proxy/application-proxy-faq.yml

```diff
@@ -246,7 +246,7 @@ sections:
               Refer to [Enable remote access to SharePoint with Azure AD Application Proxy](../manage-apps/application-proxy-integrate-with-sharepoint-server.md).
           
       - question: |
-         Can I use the SharePoint mobile app (iOS/ Android) to access a published SharePoint server?
+         Can I use the SharePoint mobile app (iOS/ Android) to access a published SharePoint Server?
```

Comment:
@kenwithee Would it be accurate to replace "to access a published SharePoint Server" with one of these options (I don't know what "published" means in this instance):

- "to access SharePoint [or SharePoint Server] hosted on-premises"
- "to access a published server running SharePoint [or SharePoint Server]"

Reply:
I think either is fine, @sassdawe  what do you think would be clearest?